src/components/BorrowerProfile/LendCta.vue

```diff
@@ -45,42 +45,88 @@
 						{{ lgScreenheadline }}
 					</p>
 					<span class="tw-flex tw-pb-1 lg:tw-pb-3">
-						<label for="LoanAmountDropdown" class="tw-sr-only">Lend amount</label>
-						<kv-ui-select
-							v-if="hideShowLendDropdown"
-							id="LoanAmountDropdown"
-							class="tw-pr-2.5 tw--mb-2"
-							v-model="selectedOption"
-							v-kv-track-event="[
-								'Lending',
-								'click-Modify loan amount',
-								'open dialog'
-							]"
-							@change="trackLendAmountSelection"
-						>
-							<option
-								v-for="priceOption in prices"
-								:key="priceOption"
-								:value="priceOption"
-							>
-								${{ priceOption }}
-							</option>
-						</kv-ui-select>
+						<form @submit.prevent="addToBasket" class="tw-w-full tw-flex">
+							<fieldset class="tw-w-full tw-flex">
```

Comment:
repeating these width and flex classes was required to ensure the new containing elements stretched to the full width of the span...